ci/pingcap_tidb_operator_build_kind.groovy

```diff
@@ -48,54 +37,29 @@ def call(BUILD_BRANCH, CREDENTIALS_ID) {
 							sudo yum install -y mercurial
 						fi
 						hg --version
+						make check-setup
+						make check
+						make test
```

Comment:
check & verify

note that test coverage for master branch will be added in another PR (https://github.com/pingcap/tidb-operator/issues/1171)